src/model/bpmn/shape/ShapeBpmnElement.ts

```diff
@@ -63,3 +64,9 @@ export class ShapeBpmnBoundaryEvent extends ShapeBpmnEvent {
 export class Participant {
   constructor(readonly id: string, readonly name?: string, public processRef?: string) {}
 }
+
+export class ShapeBpmnLane extends ShapeBpmnElement {
```

Comment:
Why do you create a new class **ShapeBpmnLane** ? 
The field **childLaneSet** is never used.

Reply:
its been used only after instantiation in buildLaneBpmnElements
I refactored it as indeed we do not need it 👍